examples/flagging-bad-clients/README.md

```diff
@@ -14,10 +14,13 @@ To each redirected message we add user properties to identify bad clients and to
 - [Docker compose](https://docs.docker.com/compose/) 
 - A HiveMQ license with enabled Data Governance Hub (contact [datagovernancehub@hivemq.com](mailto:datagovernancehub@hivemq.com))
 - A [HiveMQ Enterprise Extension for PostgreSQL](https://www.hivemq.com/extension/postgresql-extension/) license. If no license is provided, a trial license is valid for 5 hours.
-Please add your license according the [Dockerfile](hivemq/Dockerfile) 
+Please add your license according the [Dockerfile](hivemq/Dockerfile)-
```

Comment:
Shouldn't this link to the docker-compose.yml? The license addition is commented out there. 
In the Dockerfile itself, no license for DGH is mentioned.